android/app/src/main/java/com/bounswe2020group3/paperlayer/search/SearchPresenter.kt

```diff
@@ -40,7 +42,7 @@ class SearchPresenter @Inject constructor(private var model:SearchContract.Model
     override fun navigateToUser(userID: Int) {
         val bundle = bundleOf("userID" to userID )
         //FIX
```

Comment:
Can you delete this comment. You already fix it.

Reply:
I've deleted it